### PR TITLE
Helm: default tokens_file_path in ingester and store gateway

### LINKED ISSUE #2498: Set default `tokens_file_path` in helm
Setting `tokens_file_path` to ingesters and store-gateways means that there won't be series/blocks resharding on each restart. This should reduce the spikiness of memory usage of the two components.

### PR DESCRIPTION
**Pull request by andyasp**
#### What this PR does
Sets the default of `tokens_file_path` to `/data/tokens` in Helm.

First helm issue, so double check me accordingly. Tested with a local install the change was picked up and the file was used.

#### Which issue(s) this PR fixes or relates to

Fixes #2498

#### Checklist

- [x] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by krajorama — APPROVED** [review]
LGTM. I've added a non trivial commit to not ignore tokens_file_path in the helm-jsonnet comparision. Need another reviewer

File: operations/compare-helm-with-jsonnet/components/config/kustomization.yaml [hunk 1]
```diff
@@ -68,3 +68,18 @@ patches:
     patch: |-
       - op: remove
         path: /config/memberlist
+
+  # Ignore tokens_file_path, except in ingester and store gateway
```
Comment (dimitarvdimitrov) [inline operations/compare-helm-with-jsonnet/components/config/kustomization.yaml:72]: This comment is a bit confusing. It took me some time to understand that it refers to the combination of the next two replace items. At the same time it doesn't really say much more than what's already below. Can we remove it?

```suggestion
```

**Review by dimitarvdimitrov — APPROVED** [review]
LGTM, only a nitpick

**Comment by krajorama** [comment]
Trivial rebase + move changelog entry to unreleased part

**Comment by dimitarvdimitrov** [comment]
I can see this comment being posted on a new PR that's related to the jsonnet<>helm diffing https://github.com/grafana/mimir/pull/2750#issuecomment-1217860251

@andyasp can you take a look?

**Comment by dimitarvdimitrov** [comment]
it was fixed by https://github.com/grafana/mimir/pull/2753

**Pull request merged**